### DISCUSSION TITLE
Rendering inline styles. How to.

### DISCUSSION THREAD

**Discussion by wilsonowilson** [vercel/satori#324] [Q&A]
I'm using Satori to build an image generation feature for an app in production, and it's been nothing short of incredible to work with! Now I'm exploring more advanced use-cases for it.

According to the docs only `display: flex` and `display: none` are supported.

But in the future, I'd really like to include really basic inline styling in my rendered images (bold, italic, underline, code etc), like in this picture:

<img width="548" alt="CleanShot 2022-11-28 at 12 48 41@2x" src="https://user-images.githubusercontent.com/63465656/204270664-be60a2b6-e80e-4d41-a130-e5ead4bd00c6.png">

Is there a current workaround for this? Or is this something that might be implemented in the future?

**Comment by shuding** [comment]
Yep this will require inline box modals (`inline` or `inline-flex`) which we don't support for now. I will keep this open as a feature request 🙏

**Comment by Dhravya** [comment]
has there been any update to this? Any way we can make it work (I'm ready to contribute!!)

**Reply by shuding** [reply]
It's tracking in https://github.com/vercel/satori/issues/325.
